sycl/test-e2e/VirtualFunctions/2/1/1/missing-overrides.cpp

```diff
@@ -1,5 +1,4 @@
-// UNSUPPORTED: cuda, hip, acc
-// FIXME: replace unsupported with an aspect check once we have it
+// REQUIRES: aspect-ext_oneapi_virtual_functions
```

Comment:
We should move those into `lit.local.cfg` similar to what I'm doing here: https://github.com/intel/llvm/pull/15067/commits/2a73588a198fd6402b39f4980571ec3108ceffa6

Reply:
Done b457119